frontend/src/scenes/scenes.ts

```diff
@@ -210,12 +210,17 @@ export const sceneConfigurations: Partial<Record<Scene, SceneConfig>> = {
     },
     [Scene.Ingestion]: {
         projectBased: true,
-        plain: true,
+        plain: 2,
```

Comment:
Can we add an enum for the different states?

Reply:
Hmm, enum might be overkill, but made this into variants. Take a look now